### ISSUE TITLE
Amharic - Selected Site on Map

### ISSUE THREAD

**Issue by steesdale** [instedd/facility-public-portal#92]
What Happened: 
- I chose Amharic as the language
- I selected an Individual Site on the map
- I saw an error message (see below) 

![screen shot 2016-11-07 at 8 48 57 am](https://cloud.githubusercontent.com/assets/3987608/20061816/0d08f3b6-a4c7-11e6-9b79-2d885e8ee451.png)

What Was Expected: 
- I would see the site details
- I would see the portal presenting in Amharic
- The data would remain in its english upload format

**Comment by nditada** [comment]
@juanedi @vtiffenberg is this because we are missing the translated strings? Can you upload this to lang.instedd.org so we can get that translated?

**Comment by juanedi** [comment]
Fixed in STG! We had indexed an old input dataset with missing translations.

@nditada We are not using lang.instedd.org, since for the moment we assume translations are provided in the dataset itself. This is less flexible for managing one particular deployment but probably makes the project more portable for other people trying to use it.

Should we think about adding another i18n engine besides translations provided in the dataset?
